src/java/org/apache/cassandra/cql3/statements/DescribeStatement.java

```diff
@@ -132,8 +132,13 @@ public final ResultMessage execute(QueryState state, QueryOptions options, long
     @Override
     public ResultMessage executeLocally(QueryState state, QueryOptions options)
     {
-        Keyspaces keyspaces = Schema.instance.distributedAndLocalKeyspaces();
-        UUID schemaVersion = Schema.instance.getVersion();
+        UUID schemaVersion;
+        Keyspaces keyspaces;
+        do
+        {
+            schemaVersion = Schema.instance.getVersionSync();
+            keyspaces = Schema.instance.distributedAndLocalKeyspaces();
```

Comment:
@jacek-lewandowski this seems interesting at first sight ... feels unnatural. Basically, we set `keyspaces` but they are not used in `while`. Is that `distributedAndLocalKeyspaces` doing something which will make `getVersionSync` behave differently? Just asking ...

Reply:
Ah never mind ... I was reading this change wrongly. `keyspaces` are used after this do-while ... Forget it.